main.go

```diff
@@ -363,6 +371,89 @@ func getManager(
 		os.Exit(1)
 	}
 
+	discoveryClient := discovery.NewDiscoveryClientForConfigOrDie(managedCfg)
+
+	serverVersion, err := discoveryClient.ServerVersion()
+	if err != nil {
+		log.Error(err, "unable to detect the managed cluster's Kubernetes version")
+		os.Exit(1)
+	}
+
+	// use config check
+	configChecker, err := addonutils.NewConfigChecker(
+		"governance-policy-framework-addon", tool.Options.HubConfigFilePathName,
+	)
+	if err != nil {
+		log.Error(err, "unable to setup a configChecker")
+		os.Exit(1)
+	}
+
+	healthCheck := configChecker.Check
+
+	// Gatekeeper is not supported on such old versions of Kubernetes, so just always disable it in this case.
```

Comment:
I think this comment is confusing, since we're not conditionally *disabling* it, we're conditionally *enabling* it.
```suggestion
	// Only enable the GatekeeperConstraintReconciler if the Kubernetes version is v1.16 or above
```

Reply:
I think I'll remove this semver check and just assume if Gatekeeper is installed, it is working.